phpmyfaq/src/phpMyFAQ/Faq/FaqImport.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+/**
+ * Class for importing records from a csv file.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * @package   phpMyFAQ
+ * @author    Jan Harms <model_railroader@gmx-topmail.de>
+ * @copyright 2024 phpMyFAQ Team
+ * @license   https://www.mozilla.org/MPL/2.0/ Mozilla Public License Version 2.0
+ * @link      https://www.phpmyfaq.de
+ * @since     2024-01-05
+ */
+
+namespace phpMyFAQ\Faq;
+
+use phpMyFAQ\Language;
+use phpMyFAQ\Faq;
+use phpMyFAQ\Category;
+use phpMyFAQ\Configuration;
+use phpMyFAQ\Entity\FaqEntity;
+use phpMyFAQ\User\CurrentUser;
+use phpMyFAQ\Faq\FaqMetaData;
+use phpMyFAQ\Filter;
+
+/**
+ * Class FaqImport
+ *
+ * @package phpMyFAQ\Faq
+ */
+class FaqImport {
+
+    private Configuration $config;
+
+    public function __construct() {
```

Comment:
You can inject the configuration object directly in the constructor, so we don't need to assign it.